manuscript.md

```diff
@@ -2,234 +2,111 @@
+
+
+### Open Tools and Methods
+
+Machine learning encompasses a broad variety of techniques applied with or without human supervision. These techniques can often be more flexible and perform better than classical statistical methods, and can achieve a very high level of accuracy in many predictive and classification tasks in a relatively short amount of time [e.g. @Cutler2007RanFor; @Krizhevsky2017ImaCla].
+Increasing computing power combined with recent advances in machine learning techniques and applications shows promise in ecology and environmental science (see @Christin2019AppDee for an overview).
+Moreover, ongoing developments in the field of artificial intelligence are aimed at using deep learning more efficiently in low-data regimes [e.g. @Antoniou2018DatAug] and with unbalanced datasets [@Chawla2010DatMin].
+
+Machine learning is emerging as the new standard in computational ecology in general [@Olden2008MacLea; @Christin2019AppDee], and in network ecology in-particular [@Bohan2017NexGlo], as long as sufficient relevant data are available.
+Many ecological and evolutionary processes underlie species interactions and the structure of their ecological networks [e.g. @Vazquez2009UniPat; @Segar2020RolEvo].
+It can thus be difficult to choose relevant variables and model species interactions networks explicitly.
+A promising application of machine learning in natural sciences is Scientific-Machine Learning (SciML), a framework that combines machine learning with mechanistic models [@Chuang2018AdvCon; @Rackauckas2020UniDif].
+Although SciML has not yet been applied in ecology, it has the potential to combine mechanistic models with black box machine-learning to predict ecological networks with relatively small amounts of data using existing process-based models.
+Considering the current biases in network ecology [@Poisot2020EnvBia] and the scarcity of data of species interactions, the prediction of ecological networks will undoubtedly benefit from these improvements.
+
+
+### Conceptual Synthesis
+
+
+If we wish to predict the interactions between species we have not observed together, using our knowledge of the structure of ecological networks to interact in a particular ecosystem is one of our most useful assets. Other elements that contribute to the realization of an interaction are abundance and traits matching in space and time, and the combination of these elements allow us to infer potential from realized interactions and empirical data about populations
+[@Poisot2016StrPro].
+
+# A Roadmap Toward Prediction of Ecological Networks across Space and Time
+ 
+Below we focus on and discuss integrating what we envisage to be the key areas towards conceptualising and predicting ecological networks [@fig:conceptual]. With particular focus on taking a machine learning approach to the modelling process, the relationship between species interactions and network structure, and incorporating space.
```

Comment:
I added this in to try and just provide a mini roadmap of what we plan to discuss and fin a way to introduce the conceptual figure. Again not great but if we think it fits we can re-tool